src/save.js

```diff
@@ -0,0 +1,90 @@
+const Promise = require('bluebird'),
```

Comment:
Allows more advanced `Promise`, including creation of those based upon callback functions